src/web/app/tsconfig.json

```diff
@@ -13,7 +13,17 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": "./src",
+    "paths": {
+      "@components/*": ["components/*"],
+      "@hooks/*": ["hooks/*"],
+      "@styles/*": ["styles/*"],
+      "@theme": ["theme/index"],
+      "@translations/*": ["translations/*"],
+      "@config": ["config"],
+      "@interfaces": ["interfaces/index"]
+    }
```

Comment:
(nit): I personally prefer using like `@/components/*` instead of `@components/*`. It doesn't get confused if we have a package that `@components`

Reply:
I agree with @Kevan-Y here. I also use the `@/components/*` for module aliases